File: prep/average.go
package main

import "fmt"

func main() {
	fmt.Println("Calculate average")

	fmt.Println("Total numbers of scores. ")
	/* Read integer */
	var TOTAL int
    fmt.Scanf("%d", &TOTAL)
    fmt.Println("Total numbers of scores are ", TOTAL)
	
	/* Create array by name score */
    scores := make([]int, TOTAL)
    var sum = 0
    
    for i :=0; i < TOTAL; i++ {
        fmt.Println("score: ", i+1)
        fmt.Scanln(&scores[i])
        
        sum = sum + scores[i]
    }
    var avg = 0
    avg = sum / TOTAL
    fmt.Println(scores)
    fmt.Println("Total sum", sum)
    fmt.Println("average", avg)   
}

File: prep/average.py
print("Calculate average")
 
score = []
sum = 0
TOTAL = int(input("Enter number of scores :"))
for j in range(0, TOTAL):
   ele = int(input("Enter : "))
   score.append(ele)
 
for i in range(0, len(score)):
   sum = sum + score[i];   
 
avg=sum/TOTAL
print("Total number of scores", TOTAL)
print("Total Sum =", sum)
print("Average value =", avg)